dhis-2/pom.xml

```diff
@@ -546,6 +551,11 @@
         <artifactId>dhis-support-artemis</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.hisp.dhis</groupId>

```

Comment:
Remember to write a quick guide on wow on how to add Lombok to IDEs etc.

Reply:
Me and @luciano-fiandesio are already preparing a 30-60 presentation in Oslo